js/node/test/test-utils.ts

```diff
@@ -70,22 +70,24 @@ export function createTestTensor(type: Tensor.Type, lengthOrDims?: number|number
 
 // call the addon directly to make sure DLL is loaded
 export function warmup(): void {
-  describe('Warmup', async function() {
+  describe('Warmup', async function () {
     // eslint-disable-next-line no-invalid-this
     this.timeout(0);
     // we have test cases to verify correctness in other place, so do no check here.
     try {
       const session = await InferenceSession.create(path.join(TEST_DATA_ROOT, 'test_types_INT32.pb'));
-      await session.run({input: new Tensor(new Float32Array(5), [1, 5])}, {output: null}, {});
+      await session.run({ input: new Tensor(new Float32Array(5), [1, 5]) }, { output: null }, {});
     } catch (e) {
     }
   });
 }
 
 export function assertFloatEqual(
-    actual: number[]|Float32Array|Float64Array, expected: number[]|Float32Array|Float64Array): void {
-  const THRESHOLD_ABSOLUTE_ERROR = 1.0e-4;
-  const THRESHOLD_RELATIVE_ERROR = 1.000001;
+  actual: number[] | Float32Array | Float64Array, expected: number[] | Float32Array | Float64Array,
+  atol?: number, rtol?: number): void {
+
+  const absolute_tol: number = atol ?? 1.0e-4;
+  const relative_tol: number = 1 + (rtol ?? 1.0e-6);
```

Comment:
Maybe not relevant to this PR: I am curious what are these numbers (1.0e-4, 1.0e-6) for since it seems to have ATOL_DEFAULT and RTOL_DEFAULT already.